### PR TITLE
Fixed configuration profile picker

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Fixed configuration profile picker

## Changes

- Fixes: https://github.com/XRTK/XRTK-Core/issues/811

### CODE REVIEW

File: XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs [hunk 1]
```diff
@@ -19,9 +19,14 @@ namespace XRTK.Editor
     [CustomEditor(typeof(MixedRealityToolkit))]
     public class MixedRealityToolkitInspector : UnityEditor.Editor
     {
+        private const string ObjectSelectorClosed = "ObjectSelectorClosed";
```
Comment (FejZa) [inline XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: These should be camel case.

**Review by FejZa — APPROVED** [review]

File: XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs (hunk 1 above)
Comment (StephenHodgson) [reply XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: It's a const

Comment (StephenHodgson) [reply XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: Probably should be all caps but it doesn't really matter imho

Comment (FejZa) [reply XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: We recently agreed to use camel case for private const. But not too invested here. We should just stick to something I guess.

Comment (StephenHodgson) [reply XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: Mainly I just wanted it to match the exact string

Comment (StephenHodgson) [reply XRTK-Core/Packages/com.xrtk.core/Editor/MixedRealityToolkitInspector.cs:22]: Screenshot_20210419-023515_Chrome
https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants

**Pull request merged**